README.md

```diff
@@ -2,6 +2,68 @@
 
 A project that presents an iRODS Zone as HTTP v1.1.
 
+## Quickstart (Running from Docker Hub Image)
+
+Generate a local configuration JSON file.
+
+```
+docker run --rm irods/irods_http_api --dump-config-template > config.json
+```
+
+Edit/update the template `config.json` file (point to your iRODS Zone, remove OIDC and TLS sections, etc.).
+
+```
+vim config.json
+```
+
+Launch the HTTP API with your customized configuration file to check for success/errors:
```

Comment:
The statements above don't end with `:`. Perhaps this and the one below can be changed to periods too.

Do we need to say anything about checking errors here? Seems we can just say launch the http server and that's it. If there are errors, the server will yell really loud assuming it's a startup issue.

Reply:
changed two colons to periods.

leaving note about success/errors because they're never going to get it right on the first try.